lib/modules/platform/codecommit/codecommit-client.ts

```diff
@@ -54,23 +48,17 @@ import {
   UpdatePullRequestTitleInput,
   UpdatePullRequestTitleOutput,
 } from '@aws-sdk/client-codecommit';
-import type { Credentials } from '@aws-sdk/types';
+import type { RepositoryMetadata } from '@aws-sdk/client-codecommit/dist-types/models/models_0';
 import is from '@sindresorhus/is';
 import * as aws4 from 'aws4';
 import { REPOSITORY_UNINITIATED } from '../../../constants/error-messages';
 import { logger } from '../../../logger';
 
 let codeCommitClient: CodeCommitClient;
 
-export function buildCodeCommitClient(
-  region: string,
-  credentials: Credentials
-): void {
+export function buildCodeCommitClient(): void {
   if (!codeCommitClient) {
-    codeCommitClient = new CodeCommitClient({
-      region,
-      credentials,
-    });
+    codeCommitClient = new CodeCommitClient({});
```

Comment:
We now let the client detect region and credentials?

Reply:
yes. Depending on the environment.
In codebuild for example you dont need to set any params
In EC2 u need only aws region